web/multinode/src/app/store/nodes.ts

```diff
@@ -99,6 +100,18 @@ export class NodesModule implements Module<NodesState, RootState> {
         ctx.commit('populate', nodes);
     }
 
+    /**
+     * fetch action loads only online nodes information.
+     * @param ctx - context of the Vuex action.
+     */
+    public async fetchOnline(ctx: ActionContext<NodesState, RootState>): Promise<void> {
+        const nodes = ctx.state.selectedSatellite ? await this.nodes.listBySatellite(ctx.state.selectedSatellite.id) : await this.nodes.list();
+        
+        const onlineNodes = nodes.filter((node: Node) => node.statusText === NodeStatus['online']);
```

Comment:
@ifraixedes Here the `statusText` func is called.  I created this function because in the node's dropdown at the bandwidth page we only need the nodes which are online.

Reply:
Is it not possible to use `node.status` directly?

Reply:
Yes, it's possible. I removed the `statusText` method.